fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -2326,6 +2326,12 @@ struct BackupLogsDispatchTask : BackupTaskFuncBase {
 		                                             beginVersion + (CLIENT_KNOBS->BACKUP_MAX_LOG_RANGES - 1) *
 		                                                                CLIENT_KNOBS->LOG_RANGE_BLOCK_SIZE);
 
+		if (endVersion - beginVersion > CLIENT_KNOBS->BACKUP_MAX_VERSION_DIFF) {
+			TraceEvent(SevError, "BackupSawHugeVersionJump")
```

Comment:
Is there any test to verify this code path?

Reply:
I vaguely remembered FDB has an API to bump up a cluster's version. We can bump a cluster's version in the backup test workload to trigger this logic. 
@alecgrieser @sfc-gh-tclinkenbeard  do you remember the API to bump up a cluster's version?